src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,51 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $calendar-width;
+  margin: 0 auto;
```

Comment:
Don't add margin to a block, only for element. Use mix.
 More details in video.
[video, timing 4:15](https://mate.academy/learn/css-advanced/bem#/video/911)